### PR TITLE
DEVX-2314: Update all links from GH repos to docs.confluent.io: full path has changed with docs migration 

### PR DESCRIPTION
**Pull request by awalther28**
### Description 

https://confluentinc.atlassian.net/browse/DEVX-2314
_What behavior does this PR change, and why?_
Update links to map to full path in docs. I found the docs links by using grep -R " https://docs.confluent.io/current" .. Please note if you think that is not sufficient.

### Author Validation

_Describe the validation already done, or needs to be done, by the PR submitter._

- [x] Documentation --> in `utils/ccloud-generate-cp-configs.sh`

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._

- [ ] Documentation --> in `utils/ccloud-generate-cp-configs.sh`

### CODE REVIEW

File: utils/ccloud-generate-cp-configs.sh [hunk 1]
```diff
@@ -44,7 +44,7 @@
 #
 # Documentation for using this script:
 #
-#   https://docs.confluent.io/current/cloud/connect/auto-generate-configs.html
+#   https://docs.confluent.io/cloud/current/cp-component/auto-generate-configs.html#auto-generate-configs
```
Comment (ybyzek) [inline utils/ccloud-generate-cp-configs.sh:47]:
```suggestion
#   https://docs.confluent.io/cloud/current/cp-component/auto-generate-configs.html
```

File: utils/ccloud-generate-cp-configs.sh [hunk 2]
```diff
@@ -75,7 +75,7 @@ if [[ -z "$CONFIG_FILE" ]]; then
 fi
 if [[ ! -f "$CONFIG_FILE" ]]; then
   echo "File $CONFIG_FILE is not found.  Please create this properties file to connect to your Confluent Cloud cluster and then try again"
-  echo "See https://docs.confluent.io/current/cloud/connect/auto-generate-configs.html for more information"
+  echo "See https://docs.confluent.io/cloud/current/cp-component/auto-generate-configs.html#auto-generate-configs for more information"
```
Comment (ybyzek) [inline utils/ccloud-generate-cp-configs.sh:78]:
```suggestion
  echo "See https://docs.confluent.io/cloud/current/cp-component/auto-generate-configs.html for more information"
```

File: utils/ccloud-generate-cp-configs.sh [hunk 3]
```diff
@@ -86,7 +86,7 @@ if [[ -z "$SR_CONFIG_FILE" ]]; then
 fi
 if [[ ! -f "$SR_CONFIG_FILE" ]]; then
   echo "File $SR_CONFIG_FILE is not found.  Please create this properties file to connect to your Schema Registry and then try again"
-  echo "See https://docs.confluent.io/current/cloud/connect/auto-generate-configs.html for more information"
+  echo "See https://docs.confluent.io/cloud/current/cp-component/auto-generate-configs.html#auto-generate-configs for more information"
```
Comment (ybyzek) [inline utils/ccloud-generate-cp-configs.sh:89]:
```suggestion
  echo "See https://docs.confluent.io/cloud/current/cp-component/auto-generate-configs.html for more information"
```

**Review by ybyzek — COMMENTED** [review]
@awalther28 should these instances be covered in this PR as well, or separate PR?

```
README.md:You can find the documentation and instructions for this repo at [https://docs.confluent.io/current/tutorials/build-your-own-demos.html](https://docs.confluent.io/current/tutorials/build-your-own-demos.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.cp-all-in-one)
cp-all-in-one/README.md:You can find the documentation and instructions for this repo at [https://docs.confluent.io/current/tutorials/build-your-own-demos.html](https://docs.confluent.io/current/tutorials/build-your-own-demos.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.cp-all-in-one)
cp-all-in-one-cloud/README.md:You can find the documentation and instructions for this repo at [https://docs.confluent.io/current/tutorials/build-your-own-demos.html](https://docs.confluent.io/current/tutorials/build-your-own-demos.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.cp-all-in-one)
cp-all-in-one-community/README.md:You can find the documentation and instructions for this repo at [https://docs.confluent.io/current/tutorials/build-your-own-demos.html](https://docs.confluent.io/current/tutorials/build-your-own-demos.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.cp-all-in-one)
```

**Comment by awalther28** [comment]
> should these instances be covered in this PR as well, or separate PR?
> 
> ```
> README.md:You can find the documentation and instructions for this repo at [https://docs.confluent.io/current/tutorials/build-your-own-demos.html](https://docs.confluent.io/current/tutorials/build-your-own-demos.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.cp-all-in-one)
> cp-all-in-one/README.md:You can find the documentation and instructions for this repo at [https://docs.confluent.io/current/tutorials/build-your-own-demos.html](https://docs.confluent.io/current/tutorials/build-your-own-demos.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.cp-all-in-one)
> cp-all-in-one-cloud/README.md:You can find the documentation and instructions for this repo at [https://docs.confluent.io/current/tutorials/build-your-own-demos.html](https://docs.confluent.io/current/tutorials/build-your-own-demos.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.cp-all-in-one)
> cp-all-in-one-community/README.md:You can find the documentation and instructions for this repo at [https://docs.confluent.io/current/tutorials/build-your-own-demos.html](https://docs.confluent.io/current/tutorials/build-your-own-demos.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.cp-all-in-one)
> ```

@ybyzek  This PR, will push up changes shortly!

**Comment by awalther28** [comment]
@ybyzek I have updated the READMEs you pointed out.

**Review by ybyzek — APPROVED** [review]
@awalther28 LGTM!  Thank you

**Pull request merged**